src/index.html

```diff
@@ -11,7 +11,144 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post" onsubmit="onSubmit()">
+      <fieldset class="info-sections">
+        <legend>Personal information</legend>
+        <div class="box">
+          <label for="surname-value">
+            Surname:
+            <input type="text" name="surname"
+            id="surname-value" autocomplete="off"
+            required>
```

Comment:
Same here - checklist item 3.
Fix it everywhere in your document.
```suggestion
            <input
              type="text" 
              name="surname"
              id="surname-value"
              autocomplete="off"
              required
            >
```